rust/noosphere/src/ffi/petname.rs

```diff
@@ -0,0 +1,47 @@
+#![allow(unused_variables)]
```

Comment:
Caution, bikeshedding here

Reply:
I have a separate branch of work going for filling out the C FFI. I'll try to address your feedback in that change.